interval.c

```diff
@@ -26,15 +26,15 @@
 #include "timelib_private.h"
 #include <math.h>
 
-timelib_rel_time *timelib_diff(timelib_time *one, timelib_time *two)
+timelib_rel_time *timelib_diff(timelib_time *one, timelib_time *two, bool absolute)
 {
 	timelib_rel_time *rt;
 	timelib_time *swp;
 	timelib_sll dst_corr = 0 ,dst_h_corr = 0, dst_m_corr = 0;
 	timelib_time one_backup, two_backup;
 
 	rt = timelib_rel_time_ctor();
-	rt->invert = 0;
+	rt->invert = absolute;
```

Comment:
@derickr agree, it's not what expected to be done to fix the behaviour. I've just update my PR with new fix.